docs/development.md

```diff
@@ -11,18 +11,25 @@ Inside Docker, we use Bazel to version, install and build dependencies and our
 own code. Dependencies are listed in [`WORKSPACE`](/WORKSPACE). See
 https://docs.bazel.build/versions/master/external.html for more information.
 
-- Docker: https://docs.docker.com/install
-- Bazel: https://docs.bazel.build/versions/master/install.html
-- Rust: https://rustup.rs/
-  - `curl https://sh.rustup.rs -sSf > /tmp/rustup`
-  - `less /tmp/rustup` (inspect downloaded script before running it)
-  - `sh /tmp/rustup` (follow on-screen instructions -- option 1 is fine to start
+- Docker: 
+  - Follow install instructions in https://docs.docker.com/install
+- Bazel: 
+  - Follow install instructions in https://docs.bazel.build/versions/master/install.html
+- Rust: 
+  - Install instructions from https://rustup.rs/:
+    - `curl https://sh.rustup.rs -sSf > /tmp/rustup`
+    - `less /tmp/rustup` (inspect downloaded script before running it)
+    - `sh /tmp/rustup` (follow on-screen instructions -- option 1 is fine to start
     with)
-  - add `source $HOME/.cargo/env` to your shell init script (e.g. `.bashrc` or
+    - add `source $HOME/.cargo/env` to your shell init script (e.g. `.bashrc` or
     `.zshrc`)
-  - `rustup target add wasm32-unknown-unknown`
+  - Add WebAssembly target to be able to compile to WebAssembly 
+  (see [Rust Platform Support](https://forge.rust-lang.org/release/platform-support.html)):
+    - `rustup target add wasm32-unknown-unknown`
+- Protocol Buffers:
+  - Install protobuf-compiler: `apt install protobuf-compiler`
```

Comment:
I think this installs a random version which may or may not be what we use in CI, but I guess that applies to everything else in here. Maybe mention something about this at the beginning of this list, or in the previous paragraph about Dockerfile (i.e. that only the instructions in the Dockerfile contain the accurate versions used, these manual steps are just an approximation).